extensions/rich_text_components/rte-output-display.component.ts

```diff
@@ -184,6 +184,18 @@ export class RteOutputDisplayComponent implements AfterViewInit {
        * cycle and hence, we will still have the problem.
        */
       this.show = false;
+      // The text node we inserted outside angulars influence is causing some
+      // leaks. So we are manually removing the text nodes we attached.
+      const textNodes: Text[] = [];
+
+      for (const node of this.elementRef.nativeElement.childNodes) {
+        if ((node as Node).nodeType === 3) {
```

Comment:
Explain what "nodeType of 3" means.

Reply:
My bad here. To test it I used 3 which is the number for textnodes. @ashutoshc8101 I think you can do this instead:
`(node as Node).nodeType === Node.TEXT_NODE`

Reply:
Done